client/src/components/__tests__/Button.test.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import { test, expect, jest } from "@jest/globals";
+import { Button } from "../Button";
+import userEvent from "@testing-library/user-event";
+
+test("displays value as button label", () => {
+  // Arrange
+  const value = "Some Label";
```

Comment:
Lets make the props for the button globally accessible (throughout the file) so we don't have to keep defining it over and over again. (i.e. `callbackMock`, `value` and `className` should ideally only be defined once and then reused wherever it needs to be reused)